src/backend/generator.js

```diff
@@ -248,6 +248,7 @@ exports.createDistDir = function(req, socket, callback) {
       fs.remove(distHelper.distPath + '/' + appFolder, (err) => {
         if (err !== null) {
           logger.addLog('Error', 'Failed to clean up the previously existing temporary folders', socket, err);
+          callback(null);
```

Comment:
It looks like there is a missing `return` here. (Not from this PR, but from the original code.)

At present it will log an error "Failed to clean up" but then log "Successfully cleaned up" and continue with the process.

Reply:
Yes, `return done(err)` should be added here.

Reply:
Added, thanks :)